x/metadata/client/cli/tx.go

```diff
@@ -1352,27 +1339,20 @@ func validateAccAddress(addr, argName string) (string, error) {
 	return addr, nil
 }
 
-// ParseNetAssetValueString splits string (example address1,perm1,perm2...;address2, perm1...) to list of NetAssetValue's
+// ParseNetAssetValueString splits string (example 1hotdog,2jackthecat) to list of NetAssetValue's
 func ParseNetAssetValueString(netAssetValuesString string) ([]types.NetAssetValue, error) {
-	navs := strings.Split(netAssetValuesString, ";")
+	navs := strings.Split(netAssetValuesString, ",")
 	if len(navs) == 1 && len(navs[0]) == 0 {
 		return []types.NetAssetValue{}, nil
 	}
 	netAssetValues := make([]types.NetAssetValue, len(navs))
 	for i, nav := range navs {
-		parts := strings.Split(nav, ",")
-		if len(parts) != 2 {
-			return []types.NetAssetValue{}, errors.New("invalid net asset value, expected coin,volume")
-		}
-		coin, err := sdk.ParseCoinNormalized(parts[0])
-		if err != nil {
-			return []types.NetAssetValue{}, fmt.Errorf("invalid coin %s", parts[0])
-		}
-		volume, err := strconv.ParseUint(parts[1], 10, 64)
+
+		coin, err := sdk.ParseCoinNormalized(nav)
 		if err != nil {
-			return []types.NetAssetValue{}, fmt.Errorf("invalid volume %s", parts[1])
+			return []types.NetAssetValue{}, fmt.Errorf("invalid coin %s", nav)
```

Comment:
One way to help clear this up is by including what the coin is invalid for such as:

"invalid net asset value coin %s".

It doesn't have to be that, but just improves readability.